hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/checks/VerifyGetStakersNotSupported.java

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright (C) 2020-2024 Hedera Hashgraph, LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.hedera.services.bdd.spec.utilops.checks;
-
-import static com.hedera.services.bdd.spec.HapiPropertySource.asAccount;
-import static com.hederahashgraph.api.proto.java.HederaFunctionality.CryptoGetStakers;
```

Comment:
Same here and two other files

Reply:
The deleted items are no longer present in the protobuf at all.
I think the functionality is there, but the query is not.